### PR TITLE
add sonichigo

### LINKED ISSUE #584: Animesh
Screenshot

### PR DESCRIPTION
**Pull request by Sonichigo**
resolve #584

### CODE REVIEW

**Review by kaiwalyakoparkar — APPROVED** [review]
LGTM :rocket: Thanks for collaborating and contributing

File: data.json [hunk 1]
```diff
@@ -1090,5 +1090,11 @@
     "githubUsername": "hanshulll",
     "imageUrl": "https://user-images.githubusercontent.com/56799372/124683406-90eb8600-deea-11eb-9de0-d4536ed04eeb.png",
     "issueNumber": 578
+  },
+    {
```
Comment (schmelto) [inline data.json:1094]: Those should align with the other brackets

**Review by schmelto — CHANGES_REQUESTED** [review]

**Review by Sonichigo — COMMENTED** [review]
Done please check!!

**Review by schmelto — APPROVED** [review]
Lgtm

**Comment by schmelto** [comment]
Currnelty in Holliday but I have to check why the action is failing. Or someone else could do this :D

**Comment by Sonichigo** [comment]
This happened after aligning the brackets only! I guess at some point there is no alignment!

**Pull request merged**